win32ss/gdi/ntgdi/freetype.c

```diff
@@ -4217,6 +4217,10 @@ IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG lfWidth, LONG lfHeight)
         Width64 = 0;
 #endif
```

Comment:
While here, common parts could be moved outside `#if`, to emphasize what is actually different.

Reply:
Fixed. Thanks.

Reply:
Note:
`if (lfWidth != 0)` should be common too.